src/frontend/src/utils/flowResult.ts

```diff
@@ -94,7 +94,7 @@ export const apiResultToLoginFlowResult = (
     case "cancelOrTimeout": {
       return {
         tag: "err",
-        ...webAuthnCancelTemplate(),
+        ...webAuthnErrorCancelCopy(),
```

Comment:
something is a little odd, since this is probably end up being display somewhere where `displayCancelError` could be used instead; happy to fix this up later though